### PR TITLE
Small fixes to unblock publishing

### PR DESCRIPTION
**Pull request by ajpal**
Running into some issues trying to publish a new version:

Before
image

After
image

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -26,7 +26,7 @@
     "lint:test": "eslint test/unit/**",
     "start": "http-server -o",
     "pretest": "npm run lint",
-    "test": "mocha-phantomjs test/index.html",
+    "test": "mocha-phantomjs -p /usr/local/bin/phantomjs test/index.html",
```
Comment (madelynkasula) [inline package.json:29]: why is this change necessary? i'm not sure we can assume that everyone's phantomjs is at the same path. i'd be interested if @epeach has issues running this command locally since she's on linux

Comment (ajpal) [reply package.json:29]: I got this from https://github.com/nathanboktae/mocha-phantomjs/issues/217

Comment (madelynkasula) [reply package.json:29]: another option i should've mentioned -- instead of hardcoding `-p`, dynamically find the path to phantomjs (thinking of something like running `which phantomjs` from the command line, which will tell you where phantomjs is installed)

**Review by madelynkasula — APPROVED** [review]
